### DISCUSSION TITLE
关于可能出现的验证码问题

### DISCUSSION THREAD

**Discussion by Azrael070** [mikumifa/cppTickerBuy#32] [提问专区]
有传言说 cpp 这两天新接入了一些接口验证（如极验）等的消息
如果明天真遇到这个情况的话，程序会直接罢工吗？目前有考虑解决或设法绕过验证吗？

**Comment by mikumifa** [comment]
看公告，没法解决，

**Comment by mikumifa** [comment]
一段时间之后可以解决，然后用到蹲票上面

第一次出现脚本肯定救不了

**Reply by Azrael070** [reply]
明白了，感谢
